main.py

```diff
@@ -81,6 +81,27 @@ def handle_dnf(dictionary):
         output += f'{dnfer_row.name} now has {dnfer_row.dnf_count} dnfs\n'
     return output
 
+LAME_THRESHOLD = 5
+def handle_lames(dictionary):
+    # pull up all users
+    # lames are users that are in set of all users
+    # but not in (input) dictionary
+    participants = dictionary.values()
```

Comment:
isn't dictionary.values() a list of lists?

Reply:
(this is cause I shouldn't have stored dictionary as time: array of people lol

Reply:
No clue, maybe we should start using Python's type system lmao https://docs.python.org/3/library/typing.html

Reply:
LOL yeah maybe

Reply:
you can resolve this by using the keys of the `name_to_time` dict once I merge #2

Reply:
dank, I'll just rebase onto your branch